### PR TITLE
[INLONG-4439][Manager] Remove gson adapter

### LINKED ISSUE #4439: [Improve][Manager] Refactor gson adapter for managerctl
### Description

Refactor gson adapter for managerctl

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by haifxu**
### Prepare a Pull Request

- Fixes #4439 

### Motivation

When there are a lot of special fields, too many adapters need to be added, so now give up using gson and use jackson.

### Modifications

1. Remove all adapters and use Jackson instead of Gson.

2. Add an annotation to parse the status.

3.Adjust some parameters and descriptions.

### CODE REVIEW

File: inlong-manager/manager-client-tools/src/main/java/org/apache/inlong/manager/client/cli/CreateCommand.java [hunk 1]
```diff
@@ -64,8 +65,9 @@ void run() {
                     System.out.println("Create group failed: file was empty!");
                     return;
                 }
-
-                CreateGroupConf groupConf = GsonUtils.GSON.fromJson(fileContent, CreateGroupConf.class);
+                ObjectMapper objectMapper = new ObjectMapper();
```
Comment (healchow) [inline inlong-manager/manager-client-tools/src/main/java/org/apache/inlong/manager/client/cli/CreateCommand.java:68]: Please use a global instance of `ObjectMapper`.

**Review by healchow — APPROVED** [review]
LGTM

File: inlong-manager/manager-client-tools/src/main/java/org/apache/inlong/manager/client/cli/util/PrintUtils.java [hunk 2]
```diff
@@ -140,4 +149,20 @@ private static int getSpecialCharNum(String str) {
         int i = str.getBytes().length - str.length();
         return i / 2;
     }
+
+    private static <T> void parseStatus(T t) {
```
Comment (healchow) [inline inlong-manager/manager-client-tools/src/main/java/org/apache/inlong/manager/client/cli/util/PrintUtils.java:153]: Add Java doc, please.

**Review by healchow — CHANGES_REQUESTED** [review]

**Review by dockerzhang — APPROVED** [review]

**Comment by haifxu** [comment]
> @haifxu I added some Java docs for PrintUtils, please check.

ACK

**Review by healchow — APPROVED** [review]

**Pull request merged**